charts/argo/crds/cron-workflow-crd.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I think the other CRDs have an `if` here on the installCRD value in order to disable this for those who don't like helm CRD hooks